app/src/main/java/com/peacecorps/pcsa/policies_glossary/GlossaryFragment.java

```diff
@@ -20,7 +20,7 @@
 import java.util.List;
 
 /*
- * Glossary
+ * glossary of important words related to women safety.
```

Comment:
Unnecessary comment. One more important thing is this app is not about the safety of only women.